index.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
         "no-labels": 2,
         "no-lone-blocks": 2,
         "no-loop-func": 2,
+        "no-multiple-empty-lines": [2, { max: 1 }],
```

Comment:
FWIW, there are also [config options](http://eslint.org/docs/rules/no-multiple-empty-lines#options) for this rule, around EOF and BOF empty lines. We generally never have that issue but it might be good to codify expectations?